.github/scripts/plugin_tag.sh

```diff
@@ -1,4 +1,20 @@
 #!/bin/bash
+
+###### Information ############################################################################
+# Name:          plugin-tag.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         Retrieve the latest tag of a specific repo on Git based on certain filters
```

Comment:
What are the filters?

Reply:
@camerski improve the text to state as user parameters.